test/framework/tinkerbell.go

```diff
@@ -35,10 +35,6 @@ var requiredTinkerbellEnvVars = []string{
 	tinkerbellSSHAuthorizedKey,
 }
 
```

Comment:
Would we not need a `RequiredTinkerbellEnvVars()` that returns `requiredTinkerbellEnvVars`?
This will be needed when we eventually run on codebuild.